modules/plugin/mongodb/src/main/java/org/geotools/data/mongodb/MongoQueryCapabilities.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+
+package org.geotools.data.mongodb;
+
+import org.geotools.data.QueryCapabilities;
+import org.locationtech.jts.geom.Geometry;
+import org.opengis.feature.type.AttributeDescriptor;
+import org.opengis.filter.expression.PropertyName;
+import org.opengis.filter.sort.SortBy;
+
+/** A QueryCapabilities implementation for a MongoSource. */
+class MongoQueryCapabilities extends QueryCapabilities {
+
+    private MongoFeatureSource source;
+
+    MongoQueryCapabilities(MongoFeatureSource source) {
+        this.source = source;
+    }
+
+    @Override
+    public boolean supportsSorting(SortBy[] sortAttributes) {
+        if (super.supportsSorting(sortAttributes)) return true;
```

Comment:
Not sure about this one @taba90, even if the default implementation supports it we still want to override the default check no?

Reply:
fixed